organization/general_rules_and_guidelines.rst

```diff
@@ -0,0 +1,71 @@
+-------------------------------
+
+When you contribute to Godot, we expect that you respect our `Code of Conduct <https://godotengine.org/code-of-conduct/>`__.
+
+.. _doc_licenses:
+
+Respect intellectual property
+-----------------------------
+
+You must be mindful of the intellectual property of anything you submit.
+
+If your contribution was completely authored by you, this is not a problem. You can submit your contribution
+without reading further.
+
+However, any code, images, or ideas you took from somewhere else are "intellectual property" of the source
+you took it from.
+
+In this case, you need to check the license of the intellectual property. Some are permissive enough to be compatible
+with Godot's `MIT license <https://github.com/godotengine/godot/blob/master/LICENSE.txt>`__. You must include this
+license in your contribution.
+Examples of acceptable licenses include Apache 2.0, BSD, MIT, ISC, and MPL 2.0.
+
+However, others are not compatible with Godot's license. This includes "copy-left" licenses like GPL or LGPL,
+since these licenses effectively disallow static linking in proprietary software
+(which Godot is distributed as in most exported projects).
+This also applies for libraries that are only linked in the editor.
+
+Another example of incompatible licenses you should be aware of is "source-available" code.
+Most notably, you cannot submit code or ideas from proprietary game engines like Unreal or Unity.
+This includes using their code as a reference. We strongly recommend against reading any "source-available" code before
+making contributions to Godot, as this may impact your ability to contribute.
+
+Contribute only what you understand
+-----------------------------------
+
+Please only submit code that you understand and are prepared to explain to a maintainer.
+
+If you do not fully understand the code, please take extra care to test it rigorously, and disclose this in your
+pull request description.
+
+This especially applies if you implement the idea of another person, copy code from elsewhere, or if you use AI to
+assist you with your contribution. In all of these cases, you must disclose which part of your submission wasn't fully
+authored by you.
+
+AI assisted contributions
+-------------------------
+
+The use of AI to contribute to Godot is discouraged.
+
+We acknowledge that AI can be useful, but we are convinced that human effort results in better and more relevant
+contributions.
+
+If you do use AI, we expect that you put in effort to proofread and improve anything it generates, and that you disclose
+what you used the AI for. Maintainers are spending their personal time reviewing your code, it is your job to ensure
+that the code you submit is well-tested and functional.
+Please be respectful of their time and only submit something you have put thought and effort into.
+
+Contributions made entirely by AI are prohibited.
+
+.. note:: When this page refers to "AI" it means any LLM/generative
```

Comment:
It may be ideal to shift this note way up because AI is hinted at way sooner than this section, but I am not sure.